autoplex/data/common/jobs.py

```diff
@@ -561,51 +587,38 @@ def VASP_collect_data(
     if dirs is None:
         raise ValueError("dft_dir must be specified if collect_vasp is True")
 
-    failed_count = 0
     atoms = []
-    isol_es = {}
+    isolated_atom_energies = {}
 
     for i, val in enumerate(dirs):
         if os.path.exists(os.path.join(val, "vasprun.xml.gz")):
-            try:
-                converged = check_convergence_vasp(os.path.join(val, "vasprun.xml.gz"))
-
-                if converged:
-                    at = read(os.path.join(val, "vasprun.xml.gz"), index=":")
-                    for at_i in at:
-                        virial_list = (
-                            -voigt_6_to_full_3x3_stress(at_i.get_stress())
-                            * at_i.get_volume()
-                        )
-                        at_i.info["REF_virial"] = " ".join(
-                            map(str, virial_list.flatten())
-                        )
-                        del at_i.calc.results["stress"]
-                        at_i.arrays["REF_forces"] = at_i.calc.results["forces"]
-                        del at_i.calc.results["forces"]
-                        at_i.info["REF_energy"] = at_i.calc.results["free_energy"]
-                        del at_i.calc.results["energy"]
-                        del at_i.calc.results["free_energy"]
-                        atoms.append(at_i)
-                        at_i.info["config_type"] = config_types[i]
-                        if (
-                            at_i.info["config_type"] != "dimer"
-                            and at_i.info["config_type"] != "IsolatedAtom"
-                        ):
-                            at_i.pbc = True
-                            at_i.info["rss_group"] = rss_group
-                        else:
-                            at_i.info["gap_rss_nonperiodic"] = "T"
-
-                        if at_i.info["config_type"] == "IsolatedAtom":
-                            at_ids = at_i.get_atomic_numbers()
-                            # array_key = at_ids.tostring()
-                            isol_es[int(at_ids[0])] = at_i.info["REF_energy"]
-
-            except ValueError:
-                logging.error(f"Failed to collect number: {i}")
-                failed_count += 1
-                traceback.print_exc()
+            at = read(os.path.join(val, "vasprun.xml.gz"), index=":")
+            for at_i in at:
```

Comment:
Can you integrate this into this method or not, @YuanbinLiu ?

Reply:
I believe this part will need some time for merging and testing. Especially since `vaspoutput_2_extended_xyz` involves some data preprocessing, e.g., regularization, it could become confusing alongside the `preprocess_data` module in `rss`. So, I suggest we keep it as is for now, and we may need to unify the code for these two workflows as much as possible later.